tests/e2e/specs/shopper/checkout-payments.test.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	setCheckbox,
+	settingsPageSaveChanges,
+	verifyCheckboxIsSet,
```

Comment:
Hi @nielslange! The following functions aren't used in the code. I think it's safe to remove them:

- `setCheckbox`
- `settingsPageSaveChanges`
- `verifyCheckboxIsSet`

Reply:
Good catch! I just removed them.